source/sass/blocks/hero.scss

```
@@ -15,18 +15,28 @@
     right: 0;
     z-index: -1;
 
+    @media (min-resolution: 2dppx) {
+      background-image: linear-gradient(rgba(104, 183, 56, 0.85), rgba(104, 183, 56, 0.85)), url("../img/hero-background-image@2x.png");
+      background-size: contain;
```

```suggestion
      background-size: cover;
```
с этим значением лучше выглядит)